core/scripts/chaincli/handler/verify.go

```diff
@@ -0,0 +1,48 @@
+package handler
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+func (k *Keeper) changeToContractsDirectory() error {
+	// Get the current working directory
+	currentDir, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("failed to get current working directory: %w", err)
+	}
+
+	// Check if hardhat.config.ts exists in the current directory, return if it does
+	if _, err := os.Stat(filepath.Join(currentDir, "hardhat.config.ts")); err == nil {
+		return nil
+	}
+
+	// Command should run from core/scripts/chaincli, so we need to change directory to contracts
+	// Calculate the absolute path of the target directory
+	absPath := filepath.Join(currentDir, "../../../contracts")
```

Comment:
CC @RyanRHall, moving to the contracts directory is not that smart. we rely on relative path and assume people run the cli command from core/scripts/chaincli